scipy/stats/morestats.py

```diff
@@ -2706,17 +2706,27 @@ def circmean(samples, high=2*pi, low=0, axis=None):
     Returns
     -------
     circmean : float
+        nan value if no mean can be calculated (e.g., uniform 
+            circular distribution)
```

Comment:
Thanks for the tip how to run the test locally, Evgeni!

In your previous email you said that that in the current version of
scipy, the function is going to be reverted to the old one, and the
enhancement will be introduced in the next version. Probably I should
not care about the reversion but what about afterwards? Should I do a
clean commit then (with some additional tests)?

cheers!


On 30-07-16, Evgeni Burovski wrote:
>    In [1]scipy/stats/morestats.py:
> > @@ -2706,17 +2706,27 @@ def circmean(samples, high=2*pi, low=0, axis=None):
> >      Returns
> >      -------
> >      circmean : float
> > +        nan value if no mean can be calculated (e.g., uniform
> > +            circular distribution)
> 
>    I suspect it's this indent which throws refguide-check off. You can try
>    $ python runtests.py --refguide-check -s stats locally.
> 
>    —
>    You are receiving this because you were mentioned.
>    Reply to this email directly, [2]view it on GitHub, or [3]mute the
>    thread.
> 
> References
> 
>    1. https://github.com/scipy/scipy/pull/6424#discussion_r72884418
>    2. https://github.com/scipy/scipy/pull/6424/files/43a33de50086e0e6076be3fd31420133cbe49a15#r72884418
>    3. https://github.com/notifications/unsubscribe-auth/AK4rb6Wo-lQlPEXpMoY92RoDyZgBmuBuks5qaxLRgaJpZM4JWvKY